docs/content/credentials.md

```diff
@@ -32,3 +34,20 @@ before executing the step.
 Once the bundle finishes executing, the credentials are NOT recorded in the
 bundle instance (claim). Parameters are recorded there so that you can view them
 later using `porter instances show NAME --output json`.
+
+## Q & A
+
+### Why can't the credential source be defined in porter.yaml?
+
+The source of the credentials is specific to each installation of the bundle. An
```

Comment:
Perhaps `The source of a credential is specific...` or `The sources of credentials are specific`?